### ISSUE TITLE
simple package deployment 

### ISSUE THREAD

**Issue by xenigmax** [seqan/seqan#1793]
Since our supporting platforms and packages are expanding, I would like to automate release process a bit.
So that everybofy can do this job without any difficulties.

Simply it will automate below process.
Currently we are doing this by > 4 ppl.

1) make packages with different options in different machine.
2) uploading 1) to the web space
3) Update packages.seqan.de

This project includes platform independent server that can do 1) 2).
But It's not that hard to make simple one using Java or Python.
It also does not need to be very efficient.

If you guys think we need this, I will be working on it.
But the prioritization is yours.

**Comment by smehringer** [comment]
Great Idea. 
I don't know if it exactly fits here but since we will enable the version check functionality we need to maintain the release versions on the server. Currently there is a version.txt file that needs to be updated manually. An automated process to write the current seqan version tag (or app version tag) to the file would be nice.
I'd also be happy to help you then. :)

**Comment by h-2** [comment]
Good proposal, but I think release management will be part of the new infrastructure that @rrahn will be working on as part of CIBI? @rrahn can you shed some light on this?

@xenigmax For the release I do think that it would already help to have a shell script,  with the following features:
 1. given a a git tag, checkout the repository with the tag
 2. for every app name and for every architecture/flag build the packages

Then we can run this on every platform to easily get the packages and then just copy the files to the packages.seqan.de folder and regenerate homepage.

**Comment by xenigmax** [comment]
@h-2 Yes. If you and Rene think that current automation level is good enough,  I don't want to do this. :)

**Comment by rrahn** [comment]
Continues deployment is one milestone for next year.
This year we will setup continues integration for all supported platforms running on virtual machines. 
Based on this work, we can than add continues deployment so that hopefully, either a single person can trigger the release cycle, or even every night devel-binaries are submitted, if the build run through successfully. 
So, yes, it is what we want, but we will start working on that in fall probably winter this year.
@xenigmax @h-2 At the moment we have to live with the scripts we have. And I think that is ok, so far. 
But in general we can discuss the process of automated deployment in the future.
